front/src/components/app.jsx

```diff
@@ -81,6 +81,12 @@ import TasmotaEditPage from '../routes/integration/all/tasmota/edit-page';
 import TasmotaMqttDiscoverPage from '../routes/integration/all/tasmota/discover-mqtt';
 import TasmotaHttpDiscoverPage from '../routes/integration/all/tasmota/discover-http';
 
+// Netatmo integration
+import NetatmoDevicePage from '../routes/integration/all/netatmo/device-page';
+import NetatmoDiscoverPage from '../routes/integration/all/netatmo/discover-device';
+import NetatmoSettingPage from '../routes/integration/all/netatmo/setting';
+import EditNetatmoPage from '../routes/integration/all/netatmo/device-page/edit-page';
```

Comment:
for consistency more, maybe you can rename `EditNetamoPage` to `NetatmoEditPage`

Reply:
oops sorry, I scratched your nickname on the wording of the commit ...